### PR TITLE
fix(iscsi): shellcheck for modules.d/95iscsi

### PR DESCRIPTION
**Pull request by haraldh**
This pull request changes...

## Checklist
- [ ] I have tested it locally
- [ ] I have reviewed and updated any documentation if relevant
- [ ] I am providing new code and test(s) for it

Fixes #

### CODE REVIEW

**Review by johannbg — APPROVED** [review]

File: modules.d/95iscsi/parse-iscsiroot.sh [hunk 1]
```diff
@@ -47,13 +47,6 @@ if [ "${root%%:*}" = "iscsi" ]; then
     write_fs_tab /dev/root
 fi
 
-# If it's not empty or iscsi we don't continue
```
Comment (danimo) [inline modules.d/95iscsi/parse-iscsiroot.sh:50]: Is this change (with the check in 137) functionally equivalent or does it change behavior?

**Review by danimo — CHANGES_REQUESTED** [review]
See comment. Good otherwise.

File: modules.d/95iscsi/parse-iscsiroot.sh (hunk 1 above)
Comment (haraldh) [reply modules.d/95iscsi/parse-iscsiroot.sh:50]: I will double check again, after clearing my head with all the shellcheck

Comment (haraldh) [reply modules.d/95iscsi/parse-iscsiroot.sh]: reverted the logic checks

Comment (haraldh) [reply modules.d/95iscsi/parse-iscsiroot.sh]: opened https://github.com/dracutdevs/dracut/issues/1279

**Review by danimo — APPROVED** [review]

**Pull request merged**